### PR TITLE
Added new orbit bundle for opentelemetry

### PR DESCRIPTION
**Pull request by Rashmika998**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: opentelemetry-all/0.11.0.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+        <!-- Import Versions -->
+        <com.google.common.util.concurrent.version.range>[31.0.0, 32.0.0)</com.google.common.util.concurrent.version.range>
+        <com.google.common.io.version.range>[31.0.0, 32.0.0)</com.google.common.io.version.range>
+        <com.fasterxml.jackson.core.version.range>[2.13.0, 2.14.0)</com.fasterxml.jackson.core.version.range>
+        <io.netty.handler.ss.version.range>[4.1.0, 4.2.0)</io.netty.handler.ss.version.range>
+        <javax.annotation.version.range>[0.0.0, 1.0.0)</javax.annotation.version.range>
+        <javax.net.ssl.version.range>[0.0.0, 1.0.0)</javax.net.ssl.version.range>
+        <zipkin.reporter.range>[2.16.0, 2.17.0)</zipkin.reporter.range>
+        <zipkin.sender.okhttp3.range>[2.16.0, 2.17.0)</zipkin.sender.okhttp3.range>
```
Comment (tharindu1st) [inline opentelemetry-all/0.11.0.wso2v1/pom.xml:204]: can remove this

File: opentelemetry-all/0.11.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        <zipkin.range>[2.23.2, 2.23.3)</zipkin.range>
+        <okhttp3.version.range>[4.2.0, 4.3.0)</okhttp3.version.range>
+        <okio.version.range>[2.4.0, 2.5.0)</okio.version.range>
+
```
Comment (tharindu1st) [inline opentelemetry-all/0.11.0.wso2v1/pom.xml:208]: remove empty space.

File: opentelemetry-all/0.11.0.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        <!-- Import Versions -->
+        <com.google.common.util.concurrent.version.range>[31.0.0, 32.0.0)</com.google.common.util.concurrent.version.range>
+        <com.google.common.io.version.range>[31.0.0, 32.0.0)</com.google.common.io.version.range>
+        <com.fasterxml.jackson.core.version.range>[2.13.0, 2.14.0)</com.fasterxml.jackson.core.version.range>
+        <io.netty.handler.ss.version.range>[4.1.0, 4.2.0)</io.netty.handler.ss.version.range>
+        <javax.annotation.version.range>[0.0.0, 1.0.0)</javax.annotation.version.range>
+        <javax.net.ssl.version.range>[0.0.0, 1.0.0)</javax.net.ssl.version.range>
+        <zipkin.reporter.range>[2.16.0, 2.17.0)</zipkin.reporter.range>
+        <zipkin.sender.okhttp3.range>[2.16.0, 2.17.0)</zipkin.sender.okhttp3.range>
+        <zipkin.range>[2.23.2, 2.23.3)</zipkin.range>
```
Comment (tharindu1st) [inline opentelemetry-all/0.11.0.wso2v1/pom.xml:205]: this as well

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: opentelemetry-all/0.11.0.wso2v1/pom.xml (hunk 1 above)
Comment (Rashmika998) [reply opentelemetry-all/0.11.0.wso2v1/pom.xml]: Removed

File: opentelemetry-all/0.11.0.wso2v1/pom.xml (hunk 3 above)
Comment (Rashmika998) [reply opentelemetry-all/0.11.0.wso2v1/pom.xml]: Removed

File: opentelemetry-all/0.11.0.wso2v1/pom.xml (hunk 2 above)
Comment (Rashmika998) [reply opentelemetry-all/0.11.0.wso2v1/pom.xml]: Removed

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**